sensors/triggers/standard-k8s/starndard-k8s.go

```diff
@@ -114,22 +116,36 @@ func (k8sTrigger *StandardK8sTrigger) Execute(resource interface{}) (interface{}
 	}
 	obj.SetNamespace(namespace)
 
-	op := v1alpha1.Create
+	op := v1alpha1.Update
 	if trigger.Template.K8s.Operation != "" {
 		op = trigger.Template.K8s.Operation
 	}
 
-	switch op {
-	case v1alpha1.Create:
+	// If object not found, create the object. No need to have a separate `Create` switch case.
+	oldObj, err := k8sTrigger.namespableDynamicClient.Namespace(namespace).Get(obj.GetName(), metav1.GetOptions{})
+	if err != nil {
+		if !apierr.IsNotFound(err) {
+			return nil, err
+		}
 		return k8sTrigger.namespableDynamicClient.Namespace(namespace).Create(obj, metav1.CreateOptions{})
+	}
+
+	switch op {
 	case v1alpha1.Update:
-		if _, err := k8sTrigger.namespableDynamicClient.Namespace(namespace).Get(obj.GetName(), metav1.GetOptions{}); err != nil {
-			if !apierr.IsNotFound(err) {
-				return nil, err
-			}
-			return k8sTrigger.namespableDynamicClient.Namespace(namespace).Create(obj, metav1.CreateOptions{})
+		if err := mergo.Merge(oldObj, obj); err != nil {
```

Comment:
`mergo.Merge(oldObj, obj, mergo.WithOverride)`

Without `WithOverride`, it won't merge the non-empty fields in the destination.